Dockerfile

```diff
@@ -1,23 +1,18 @@
 # Openhim Console Dockerfile for latest changes
-FROM ubuntu:16.04
+FROM node:14.17-alpine
+RUN apk upgrade --update-cache --available
+RUN apk add git
```

Comment:
Why is this here?

Reply:
I found some dependencies in the package.json file that referenced git repositories. Without these the application would not build.

Reply:
I see, so then I guess perhaps the node alpine can be part of the multi-stage build. Then just copy over the `/dist` directory into the alpine image

Reply:
That is done